thanos/thanos.js

```diff
@@ -1,8 +1,9 @@
 let lib;
 
-const Logger = require('./js/Logger');
-const DatabaseModify = require('./js/databaseModify');
 const _ = require('lodash');
+const Logger = require('Logger'); // from shared layer
+const DatabaseModify = require('databaseModify'); // from shared layer
```

Comment:
Same comment once again.  Why are you not using the ternary syntax for these `require` statements?